src/mount/mount.ceph.c

```diff
@@ -912,7 +912,8 @@ static int do_mount(const char *dev, const char *node,
 			fprintf(stderr, "mount error: ceph filesystem not supported by the system\n");
 			break;
 		case EHOSTUNREACH:
-			fprintf(stderr, "mount error: no mds server is up or the cluster is laggy\n");
+			fprintf(stderr, "mount error: no mds server is up — "
+			"the cluster may be laggy, or you may not be authorized\n");
```

Comment:
```suggestion
			fprintf(stderr, "mount error: no MDS(Metadata Server) is up.  "
			"The cluster might be laggy or you may lack authorization \n");
```

Reply:
the reason for first line - mds server translates to "Metadata Server server" while sounds a bit weird to me